File: TakeDocApp/www/models/readOnlyMetadata.js
var ReadOnlyMetadata = Backbone.Model.extend({
    defaults: {
        Name: null,
        EntityId: null,
        Label: null,
        Value: null,
        Text: null,
        DisplayIndex: null,
        Type: null
    },
    parse: function () {
        var current = arguments[0];
        this.set("Name", current.Name);
        this.set("EntityId", current.EntityId);
        this.set("Label", current.Label);
        this.set("Value", current.Value);
        this.set("Text", current.Text);
        this.set("DisplayIndex", current.DisplayIndex);
        this.set("Type", current.Type);
        if (current.Type == "date") this.set("Text", moment(current.Text).format("L"));
        return this;
    }
});

var ReadOnlyMetadatas = Backbone.Collection.extend({
    model: ReadOnlyMetadata,
    urlBase: environnement.UrlBase + "/MetaData/",
    loadBy: function (param) {
        this.url = this.urlBase + "Version/ReadOnly/<versionId/>/<entityId/>";
        this.url = this.url.replace("<versionId/>", param.versionId).replace("<entityId/>", param.entityId);
        this.fetch({ success: param.success, error: param.error });
    },
    parse: function () {
        var data = arguments[0];
        var arr = new Array();
        for (var i = 0; i < data.length; i++) {
            var current = new ReadOnlyMetadata();
            arr.push(current.parse(data[i]));
        }
        return arr;
    },
});

File: TakeDocApi/TakeDocModel/Dto/Document/ReadOnlyMetadata.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TakeDocModel.Dto.Document
{
    public class ReadOnlyMetadata
    {
        public string Name { get; set; }
        public Guid EntityId {get; set;}
        public string Label { get; set; }
        public string Value { get; set; }
        public string Text { get; set; }
        public int DisplayIndex { get; set; }
        public string Type { get; set; }
    }
}
